### DISCUSSION TITLE
windows系统下 BMP导出 和混合导出会失败，没有报错码，矢量导出可以成功

### DISCUSSION THREAD

**Discussion by EddieHxc** [Tencent/libpag#2027] [Q&A] [answered]
![20231221-112122](https://github.com/Tencent/libpag/assets/126870582/034e4fc7-c2eb-47ba-981b-cc9946dc835c)
![20231221-112132](https://github.com/Tencent/libpag/assets/126870582/8f940610-8aeb-4b14-bc3a-1bbec84abd7f)

**Comment by EddieHxc** [comment]
系统win 11

**Comment by kevingpqi123** [comment]
这类问题建议提供出现问题的 AE 工程，另外，请不要重复提问，如果需要其它信息需要说明可以附在原贴下面

**Reply by EddieHxc** [reply]
[saberUntitled Project folder.zip](https://github.com/Tencent/libpag/files/13737419/saberUntitled.Project.folder.zip)

**Reply by EddieHxc** [reply]
好的 这个是使用的导出版本
![Uploading 20231221-153425.jpg…]()

**Reply by EddieHxc** [reply]
AE使用2020版与2024版 都有这个问题 排除了AE版本的问题

**Reply by Irlyue** [reply]
这个问题能解答下吗，我这里也出现了同样的问题。bmp导出花费很久，然后100%的时候又显示导出失败。

**Reply by EddieHxc** [reply]
插件的BUG

**Comment by createmomo** [comment]
你好，请问后来解决了嘛？我也遇到了类似的问题。

**Reply by EddieHxc** [reply]
没有， 他们并没有修复这个问题

**Comment by whyisthishard** [comment]
我也遇到过，电脑从新mac换成旧mac，成功导出，操作步骤都一致

**Comment by jinwuwu001** [comment, marked as answer]
经确认该问题的原因是PAGViewer在进行ae插件安装时没有正确安装264编码工具，该问题是偶现的，目前正在处理中，后续版本会修复，可通过如下两种方式暂时规避该问题
1、可以在PAGViewer中多次安装ae插件，确保%APPDATA%\H264EncoderTools下存在编码工具后再进行pag导出
2、在PAGViewer安装目录下找到可执行程序PAGViewer/H264EncoderTools.exe，将其复制至%APPDATA%\H264EncoderTools，然后再进行pag文件导出
